sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -191,43 +189,21 @@ class MapboxMap internal constructor(
     onStyleLoaded: Style.OnStyleLoaded? = null
   ) {
     this.style = style
-    var resourceCount = styleExtension.resourceCount
-    styleExtension.sources.forEach {
-      if (it is GeoJsonSource) {
-        it.bindTo(style)
-        it.addOnGeoJsonParsedListener(object : OnGeoJsonParsed {
-          override fun onGeoJsonParsed(source: GeoJsonSource) {
-            resourceCount--
-            if (resourceCount == 0) {
-              onStyleLoaded?.onStyleLoaded(style)
-            }
-            it.removeOnGeoJsonParsedListener(this)
-          }
-        })
-      } else {
-        it.bindTo(style)
-        resourceCount--
-      }
-    }
     styleExtension.images.forEach {
       it.bindTo(style)
-      resourceCount--
     }
-    styleExtension.layers.forEach {
-      it.first.bindTo(style, it.second)
-      resourceCount--
-    }
-    styleExtension.light?.let {
+    styleExtension.sources.forEach {
       it.bindTo(style)
-      resourceCount--
     }
-    styleExtension.terrain?.let {
+    styleExtension.images.forEach {
```

Comment:
here `styleExtension.images.forEach .. ` is already done above.

Reply:
💎 catch!
thanks!